connect/runtime/src/test/java/org/apache/kafka/connect/runtime/WorkerConnectorTest.java

```diff
@@ -120,43 +105,21 @@ public void testInitializeFailure() {
         workerConnector.doShutdown();
         assertStoppedMetric(workerConnector);
 
-        verifyAll();
+        verifyCleanInitialize();
+        verify(listener).onFailure(CONNECTOR, exception);
+        verify(listener).onShutdown(CONNECTOR);
```

Comment:
The one for shutdown makes sense but I think with startup there are three different cases - 

i) No call to `connector.start` or `listener.onStartup` (failure in initialization OR started in the paused state)
ii) Calls to both `connector.start` and `listener.onStartup` (successful start)
iii) Call to only `connector.start` (connector started in the paused state and then resumed OR failure on startup)

I think it might be more readable to keep these verify calls in the test methods as is rather than trying to force fit them into (in)appropriately named methods, WDYT?

Reply:
Yeah, that's fair. LGTM 👍